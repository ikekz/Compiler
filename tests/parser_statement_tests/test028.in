procedure p(p: integer);
begin
end;
begin
	p();
end.