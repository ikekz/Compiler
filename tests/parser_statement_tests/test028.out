(5,2) Error: Wrong number of parameters specified for call to "p"